OneSignalSDK/onesignal/core/src/main/java/com/onesignal/core/internal/operations/impl/OperationModelStore.kt

```diff
@@ -48,14 +53,15 @@ internal class OperationModelStore(prefs: IPreferencesService) : ModelStore<Oper
             SubscriptionOperationExecutor.DELETE_SUBSCRIPTION -> DeleteSubscriptionOperation()
             SubscriptionOperationExecutor.TRANSFER_SUBSCRIPTION -> TransferSubscriptionOperation()
             LoginUserOperationExecutor.LOGIN_USER -> LoginUserOperation()
+            LoginUserFromSubscriptionOperationExecutor.LOGIN_USER_FROM_SUBSCRIPTION_USER -> LoginUserFromSubscriptionOperation()
             RefreshUserOperationExecutor.REFRESH_USER -> RefreshUserOperation()
             UpdateUserOperationExecutor.SET_TAG -> SetTagOperation()
             UpdateUserOperationExecutor.DELETE_TAG -> DeleteTagOperation()
             UpdateUserOperationExecutor.SET_PROPERTY -> SetPropertyOperation()
             UpdateUserOperationExecutor.TRACK_SESSION_START -> TrackSessionStartOperation()
             UpdateUserOperationExecutor.TRACK_SESSION_END -> TrackSessionEndOperation()
             UpdateUserOperationExecutor.TRACK_PURCHASE -> TrackPurchaseOperation()
-            else -> throw Exception("Unrecognized operation: $operationName")
```

Comment:
I think we should still throw, otherwise it will make finding the root cause much harder later.

Reply:
Agreed I reverted this change